docs/simulators/DualSPHysics.md

```diff
@@ -48,6 +52,395 @@ task.download_outputs()
 
 ```
 
+Below, are going to show how to run a slighly more complex examples that come
+included with the DualPhysics distribution inside the `examples` folder. 
+In general, the procedure involves transforming the shell script that controls
+the several steps of the simulation into a list of commands that will then be
+passed via the `run()` method as shown above.
+
+# Running `examples/chrono/09_Turbine`
+We are going to run the simulation stored in `examples/chrono/09_Turbine`.
+
+![Video produced with Paraview using VTK files generated by DualSPHysics](turbine30fps.gif)
+
+Originally, this simulation is orchestrated by a shell script that receives
+parameters from the user. The script takes those parameters, performs some 
+initial configurations and then runs a series of Dualphysics commands, including
+pre-processing, simulation and a number of post-processing steps.
```

Comment:
including pre-processing, simulation, and a number of post-processing steps.

Reply:
Done.